src/lib/storage/fixed_string_dictionary_segment/fixed_string_span.hpp

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include <memory>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "fixed_string.hpp"
+#include "fixed_string_span_iterator.hpp"
+#include "types.hpp"
+#include "utils/assert.hpp"
+
+namespace hyrise {
+
+// FixedStringSpan is a data type, which stores all its values in a vector and
```

Comment:
this comment is wrong or am I mistaken? It doesn't store all its values in a vector

Reply:
Will be done by Tobias